pkgs/top-level/php-packages.nix

```diff
@@ -675,6 +675,26 @@ in
       meta.broken = isPhp74;
     };
 
+    rdkafka = buildPecl {
+      version = "4.0.3";
+      pname = "rdkafka";
+
+      sha256 = "1g00p911raxcc7n2w9pzadxaggw5c564md6hjvqfs9ip550y5x16";
+
+      buildInputs = with pkgs; [ rdkafka pcre' ];
+
+      postPhpize = ''
+        substituteInPlace configure \
+          --replace 'SEARCH_PATH="/usr/local /usr"' 'SEARCH_PATH=${pkgs.rdkafka}'
```

Comment:
Maybe a regex that looks for a line starting with SEARCH_PATH would be more robust.

Reply:
It's also more complex to read, they have had the same `SEARCH_PATH` as a result from 3.0.3 to 4.0.3, so it doesn't seem to change that much.

Also, if it would fail to build. It's easy to spot that during the package update.